src/app/families/DeliveryStatus.ts

```diff
@@ -85,6 +85,10 @@ export class DeliveryStatusColumn extends ValueListColumn<DeliveryStatus> {
   isProblem() {
     return this.isGreaterOrEqualTo(DeliveryStatus.FailedBadAddress).and(this.isLessOrEqualTo(DeliveryStatus.FailedOther));
   }
+  isNotProblem() {
+    return this.isLessOrEqualTo(DeliveryStatus.SuccessLeftThere).and(this.isDifferentFrom(DeliveryStatus.Frozen));
```

Comment:
this will include also ready for delivery and self pickup - I hope that's what you wanted

Reply:
yes. exactly what I wanted. everything that either received or still on the way.